
＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,943                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15298]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,954                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15308]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,955                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15309]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,955                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15310]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,956                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15311]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,957                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15311]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,958                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15312]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,958                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15313]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,959                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15313]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,960                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15314]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,960                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15315]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,961                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15316]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,962                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15316]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,963                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15317]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,963                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15318]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,964                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15319]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,965                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15319]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,966                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15320]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,966                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15321]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,967                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15322]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,968                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15322]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,969                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15323]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,971                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15326]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,972                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15326]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,973                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15327]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,973                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15328]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,974                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15329]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,975                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15330]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,976                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15330]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,977                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15331]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,977                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15332]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,978                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15333]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,979                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15333]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,980                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15334]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,980                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15335]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,981                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15336]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,982                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15337]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,983                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15338]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,984                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15339]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,985                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15339]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,986                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15340]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,986                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15341]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,987                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15342]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,988                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15342]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,989                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15343]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,989                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15344]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,990                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15345]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,991                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15346]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,992                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15347]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,993                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15347]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,994                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15348]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,994                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15349]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,995                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15350]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,996                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15350]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,997                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15351]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,998                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15352]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,000                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15355]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,001                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15355]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,002                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15356]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,002                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15357]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,003                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15358]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,004                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15358]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,005                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15359]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,006                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15360]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,006                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15361]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,007                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15362]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,008                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15363]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,009                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15363]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,010                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15364]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,010                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15365]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,011                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15366]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,012                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15366]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,013                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15367]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,013                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15368]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,014                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15369]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,015                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15369]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,016                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15370]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,016                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15371]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,017                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15372]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,018                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15373]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,019                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15373]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,020                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15374]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,020                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15375]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,021                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15376]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,022                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15377]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,023                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15377]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,024                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15378]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,024                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15379]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,025                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15380]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,026                                   
【线程编号】[.NET Long Running Task]                                   
【执行时间】[15381]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,931                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15285]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,029                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15383]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,029                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15384]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,030                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15384]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,030                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15385]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,031                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15385]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,031                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15386]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,032                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15386]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,032                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15387]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,033                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15387]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,033                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15387]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:44,272                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[14627]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,034                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15388]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,034                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15389]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,035                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15389]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,035                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15389]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,035                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15390]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,036                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15390]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,036                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15391]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,036                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15391]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,037                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15391]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,037                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15392]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26

＝＝＝＝＝＝＝＝＝＝                                   
【日志级别】ERROR                                   
【记录时间】2021-07-25 15:56:45,038                                   
【线程编号】[.NET ThreadPool Worker]                                   
【执行时间】[15392]毫秒                                   
【出错文件】D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\src\RDLinker.Log\LogHelper.cs                                   
【出错行号】38                                   
【出错的类】rdlinker-log-default 属性[(null)]                                   
【错误描述】i from rdlinker test log                                   
【错误详情】
System.Exception: i from rdlinker test log
   at RDLinker.Test.Log.UnitTest1.Test1() in D:\MyProjects\Git\RdLinker\Core\RDLinker.Core\test\RDLinker.Test.Log\UnitTest1.cs:line 26
